divisible.rb

```diff
@@ -0,0 +1,13 @@
+# Write a method that returns in an array all the numbers between 1 and 100 that are divisible by 2 or 3 or 5. Then call the method in your program and print out what it returns.  Call the program divisible.rb.
+
+def divisible
+    arr = []
+    (1..100).each { |num| 
+        if num % 2 == 0 || num % 3 == 0 || num % 5 == 0
+            arr.push(num)
+        end
+    }
+    puts arr.inspect()
+end
+
+divisible()
```

Comment:
The _output_ is good; however, the assignment wants the method to return an array, and [#puts](https://www.rubydoc.info/stdlib/core/IO:puts) returns `nil` instead. Could you make some small adjustments to this so that it satisfies the requirements?

Reply:
Remove "puts" in order to return the array instead of printing it as a string.